src/modules/storage/LedgerStorage.ts

```diff
@@ -50,11 +60,57 @@ export class LedgerStorage extends Storages
             cycle_length INTEGER NOT NULL,
             enroll_sig TEXT NOT NULL,
             PRIMARY KEY("block_height","enrollment_index")
-        );`;
-        this.db.each(sql, (err: Error | null) =>
+        );
+
+        CREATE TABLE IF NOT EXISTS "transactions"
+        (
+            "block_height"      INTEGER NOT NULL,
+            "tx_index"          INTEGER NOT NULL,
+            "tx_hash"           TEXT NOT NULL,
+            "type"              INTEGER NOT NULL,
+            "inputs_count"      INTEGER NOT NULL,
+            "outputs_count"     INTEGER NOT NULL,
+            PRIMARY KEY("block_height", "tx_index")
+        );
+
+        CREATE TABLE IF NOT EXISTS "tx_inputs"
+        (
+            "block_height"      INTEGER NOT NULL,
+            "tx_index"          INTEGER NOT NULL,
+            "in_index"          INTEGER NOT NULL,
+            "previous"          TEXT NOT NULL,
+            "out_index"         INTEGER NOT NULL,
+            PRIMARY KEY("block_height", "tx_index", "in_index")
+        );
+
+        CREATE TABLE IF NOT EXISTS "tx_outputs"
+        (
+            "block_height"  INTEGER NOT NULL,
+            "tx_index"      INTEGER NOT NULL,
+            "output_index"  INTEGER NOT NULL,
+            "tx_hash"       TEXT NOT NULL,
+            "utxo_key"      TEXT NOT NULL,
+            "amount"        NUMERIC NOT NULL,
+            "address"       TEXT NOT NULL,
+            "used"          INTEGER NOT NULL DEFAULT 0,
+            PRIMARY KEY("block_height", "tx_index", "output_index")
+        );
+
+        CREATE TABLE IF NOT EXISTS "validators"
+        (
+            "enrolled_at"	INTEGER NOT NULL,
+            "utxo_key"	TEXT NOT NULL,
```

Comment:
There is a mix of spaces and tabs here. I think there is a setting in vscode so you never accidentally add tabs.

Reply:
I fixed it